onyo/commands/config.py

```diff
@@ -16,15 +16,17 @@
         metavar='ARGS',
         nargs='+',
         type=git_config,
-        help='Config options to set in .onyo/config'),
+        help='Config options to set in .onyo/config'
+    ),
 }
 
 
 def config(args: argparse.Namespace) -> None:
     """
-    Set, query, and unset Onyo repository configuration options. These options
-    are stored in ``.onyo/config``. This file is tracked by git and are shared
-    with all other consumers of an Onyo repository.
+    Set, query, and unset Onyo repository configuration options.
+
+    These options are stored in ``.onyo/config``, which is is tracked by git and
```

Comment:
```suggestion
    These options are stored in ``.onyo/config``, which is tracked by git and
```

Reply:
Thanks for catching this. Fixed and pushed. :-)